src/components/context/ShareItemModalContext.js

```diff
@@ -119,44 +127,53 @@ const ShareItemModalProvider = ({ children }) => {
     navigator.clipboard.writeText(link);
```

Comment:
Actually there is a util function in `utils/clipboard.js` `copyToClipboard()` where you can also pass onSuccess and onError. It would be nice to have notification feedbacks!